Buildings/Experimental/OpenBuildingControl/CDL/Discrete/TriggeredSampler.mo

```diff
@@ -1,62 +1,35 @@
 within Buildings.Experimental.OpenBuildingControl.CDL.Discrete;
-block TriggeredSampler "Output the triggered sampled value of a continuous signal"
-
-  parameter Modelica.SIunits.Time samplePeriod(min=100*1E-15)
-    "Sample period of component";
-
-  parameter Modelica.SIunits.Time startTime=0 "First sample time instant";
+block TriggeredSampler "Triggered sampling of continuous signals"
 
   parameter Real y_start=0 "initial value of output signal";
 
   Interfaces.RealInput u "Connector with a Real input signal"
     annotation (Placement(transformation(extent={{-140,-20},{-100,20}})));
-
   Interfaces.RealOutput y "Connector with a Real output signal"
     annotation (Placement(transformation(extent={{100,-10},{120,10}})));
-
-  Interfaces.BooleanInput trigger "Connector for trigger"
-    annotation (Placement(
+  Interfaces.BooleanInput trigger annotation (Placement(
         transformation(
         origin={0,-118},
         extent={{-20,-20},{20,20}},
         rotation=90)));
-
-protected
-  output Boolean sampleTrigger "True, if sample time instant";
-
-  output Boolean firstTrigger(start=false, fixed=true)
-    "Rising edge signals first sample instant";
-
-initial equation
-  y = y_start;
-
 equation
-  // Declarations that are used for all discrete blocks
-  sampleTrigger = sample(startTime, samplePeriod);
-  when sampleTrigger then
-    firstTrigger = time <= startTime + samplePeriod/2;
-  end when;
-
-  // Declarations specific to this type of discrete block
   when trigger then
     y = u;
   end when;
-
+initial equation
```

Comment:
Please use the `initial equation` section above the `equation` section (which is what all models have I believe).